Error on space in directory path

If there is a space in the directory path the compilation will fail.

Error log:
```
Error: Cannot find module 'C:\Dev'
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Function.Module.runMain (module.js:441:10)
    at startup (node.js:139:18)
    at node.js:968:3
```

--- a/sbt-scalajs-bundler/src/main/scala/scalajsbundler/ExternalCommand.scala
+++ b/sbt-scalajs-bundler/src/main/scala/scalajsbundler/ExternalCommand.scala
@@ -18,12 +18,12 @@ class ExternalCommand(name: String) {
     * @param logger Logger
     */
   def run(args: String*)(workingDir: File, logger: Logger): Unit =
-    Commands.run((cmd +: args).mkString(" "), workingDir, logger)
+    Commands.run(cmd ++: args, workingDir, logger)
 
   val cmd =
     sys.props("os.name").toLowerCase match {
-      case os if os.contains("win") => s"cmd /c $name"
-      case _ => name
+      case os if os.contains("win") => Seq("cmd", "/c", name)
+      case _ => Seq(name)
     }
 
 }

--- a/sbt-scalajs-bundler/src/main/scala/scalajsbundler/ReloadWorkflow.scala
+++ b/sbt-scalajs-bundler/src/main/scala/scalajsbundler/ReloadWorkflow.scala
@@ -68,7 +68,7 @@ object ReloadWorkflow {
         }
       val concatFile = targetDir / "scalajsbundler-concat.js"
       IO.write(concatFile, concatContent.show)
-      Commands.run(s"node ${concatFile.absolutePath}", workingDir, logger)
+      Commands.run(Seq("node", "concatFile.absolutePath"), workingDir, logger)
     } else {
       logger.info("Bundling dependencies")
       IO.copyFile(bundledDependencies, bundle)

--- a/sbt-scalajs-bundler/src/main/scala/scalajsbundler/ReloadWorkflow.scala
+++ b/sbt-scalajs-bundler/src/main/scala/scalajsbundler/ReloadWorkflow.scala
@@ -68,7 +68,7 @@ object ReloadWorkflow {
         }
       val concatFile = targetDir / "scalajsbundler-concat.js"
       IO.write(concatFile, concatContent.show)
-      Commands.run(Seq("node", "concatFile.absolutePath"), workingDir, logger)
+      Commands.run(Seq("node", concatFile.absolutePath), workingDir, logger)
     } else {
       logger.info("Bundling dependencies")
       IO.copyFile(bundledDependencies, bundle)

--- a/sbt-scalajs-bundler/src/main/scala/scalajsbundler/Webpack.scala
+++ b/sbt-scalajs-bundler/src/main/scala/scalajsbundler/Webpack.scala
@@ -122,7 +122,7 @@ object Webpack {
     */
   def run(args: String*)(workingDir: File, log: Logger): Unit = {
     val webpackBin = workingDir / "node_modules" / "webpack" / "bin" / "webpack"
-    val cmd = s"node ${webpackBin.absolutePath} ${args.mkString(" ")}"
+    val cmd = "node" +: webpackBin.absolutePath +: args
     Commands.run(cmd, workingDir, log)
     ()
   }

--- a/sbt-scalajs-bundler/src/main/scala/scalajsbundler/util/Commands.scala
+++ b/sbt-scalajs-bundler/src/main/scala/scalajsbundler/util/Commands.scala
@@ -4,7 +4,7 @@ import sbt._
 
 object Commands {
 
-  def run(cmd: String, cwd: File, logger: Logger): Unit = {
+  def run(cmd: Seq[String], cwd: File, logger: Logger): Unit = {
     val process = Process(cmd, cwd)
     val code = process ! logger
     if (code != 0) {

